tests/integration/test_installation.py

```diff
@@ -767,16 +778,11 @@ def test_table_migration_job_cluster_override(  # pylint: disable=too-many-local
     make_dbfs_data_copy,
     sql_backend,
 ):
-    # TODO - DBFS API does not support instance profile based mounts
-    if not ws.config.is_azure:
-        pytest.skip("Temporarily does not work on AWS")
     # create external and managed tables to be migrated
     schema = make_schema(catalog_name="hive_metastore", name=f"migrate_{make_random(5).lower()}")
     src_managed_table = make_table(schema_name=schema.name)
     existing_mounted_location = f'dbfs:/mnt/{env_or_skip("TEST_MOUNT_NAME")}/a/b/c'
-    new_mounted_location = f'dbfs:/mnt/{env_or_skip("TEST_MOUNT_NAME")}/a/b/{make_random(4)}'
-    make_dbfs_data_copy(src_path=existing_mounted_location, dst_path=new_mounted_location)
```

Comment:
this will be fixed in #1403

Reply:
Cool, but even the current approach is not wrong. Is there a risk doing this way?